test/topojsonm/expected/test.ml

```diff
@@ -34,4 +33,22 @@ let test_map_objects_topojson () =
       Topojsonm.Err.pp Format.err_formatter e;
       failwith "Internal err"
 
-let () = test_map_objects_topojson ()
```

Comment:
We need to still call the `test_map_objects_topojson` function in the main `let () =` -- that's why the tests are failing in CI I think at the moment